src/callable/primitive/paste.rs

```diff
@@ -101,109 +101,87 @@ impl Callable for PrimitivePaste {
 }
 
 #[cfg(test)]
-mod test_primitive_paste {
+mod test {
     use super::*;
-    use crate::parser::parse;
+    use crate::assert_r_eq;
 
     #[test]
-    fn test_primitive_paste_01() {
-        let mut stack = CallStack::new();
-        let expr = parse("paste(1, 2, collapse = NULL)").unwrap();
-        let R::Vector(res) = stack.eval(expr).unwrap() else { unimplemented!() };
-        let observed: Vec<_> = res.into();
-        let expected: Vec<_> = vec!["1 2"];
-        assert_eq!(observed, expected);
+    fn numeric_input() {
+        assert_r_eq!(
+            R{ paste(1, 2, collapse = NULL) }, 
+            R{ "1 2" }
+        );
     }
```

Comment:
@armenic Added some test helpers! Much less boilerplate needed now to test the R interface. Hopefully this makes it easier to keep up with test writing. Thanks for being the driving force behind getting us started with more testing!